crates/extensions/c8y_mapper_ext/src/actor.rs

```diff
@@ -165,8 +175,7 @@ impl C8yMapperActor {
             .with_retain();
```

Comment:
You can use `Topic::new_unchecked("")` to create the topic, no need to worry about `unwrap`.

Reply:
okay, looks like `new_unchecked` saves an extra check, so I will be using it from now on